README.md

```diff
@@ -46,4 +46,4 @@ Please read through the [Contribution Guide](https://github.com/Codecademy/docs/
 
 ## Additional Notes
 
-Remember, if you ever have any questions at all, we're always here to help in the [Codecademy Forums](https://discuss.codecademy.com/) and [Codecademy Discord](https://discord.com/invite/codecademy).
+Remember, if you ever have any questions at all, we're always here to help in the [Codecademy Forums](https://discuss.codecademy.com/c/get-help/codecademy/1836) and [Codecademy Discord](https://discord.com/invite/codecademy).
```

Comment:
For the "Codecademy Discord" link, should we mention which channel(s) to check out after their invite is accepted?

Reply:
I would say yes. Something like:

For general questions about the docs use `#general` but for language-specific questions use the channel for that language.